mirage/unikernel.ml

```diff
@@ -100,83 +98,137 @@ module K = struct
     Mirage_runtime.register_arg Arg.(value & opt_all domain [] doc)
 
   (* DNSmasq configuration options *)
-  (* TODO support multiple dhcp-range statements *)
-  let dhcp_range =
-    let doc =
-      Arg.info ~doc:"Enable DHCP server." ~docv:Config_parser.dhcp_range_docv
-        ~docs:s_dnsmasq [ "dhcp-range" ]
-    in
-    Mirage_runtime.register_arg
+  module Dnsmasq = struct
+    let s_dnsmasq = "DNSMASQ-COMPATIBLE OPTIONS"
+
+    (* TODO support multiple dhcp-range statements *)
+    let dhcp_range =
+      let doc =
+        Arg.info ~doc:"Enable DHCP server." ~docv:Config_parser.dhcp_range_docv
+          ~docs:s_dnsmasq [ "dhcp-range"; "F" ]
```

Comment:
the `"F"`, if we want to support it, could we do this in a separate PR? I'm not too convinced by providing such abbreviated parameters...